dlc_developer_config.toml

```diff
@@ -25,19 +25,19 @@ do_build = true
 
 [test]
 sanity_tests = true
-  safety_check_test = false
+safety_check_test = true
```

Comment:
when you revert, can you add the two spaces back? it's not functional, just shows that safety check is a subset of sanity